doc/Getting_Started.md

```diff
@@ -37,24 +37,18 @@ of installing some of these tools, but you can also install them yourself.
 
 #### Rust (nightly)
 
-We are using `rustc 1.24.0-nightly (8e7a609e6 2018-01-04)`. We recommend
+We are using `rustc 1.26.0-nightly (2789b067d 2018-03-06)`. We recommend
 installing it with [rustup](http://www.rustup.rs) so you can manage multiple
 versions of Rust and continue using stable versions for other Rust code:
 
 ```bash
-$ curl https://sh.rustup.rs -sSf | sh
+$ curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly-2018-03-07
```

Comment:
I'm not sure we want to set the default toolchain for people. We try pretty hard to not mess with people's default environment. All of the build utilities set `RUSTUP_TOOLCHAIN` so that we grab the right toolchain while operating without polluting the user's environment

Reply:
Ok, makes sense I'll omit these changes!